code/script.js

```diff
@@ -1,19 +1,148 @@
-// Start here
-
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  `Welcome to our Javascript Pizzeria 🧚🏼‍♀️ 
+  Ready to Start?
+  Click 🆗 to begin!
+  `
 )
 
+const yourName = prompt("What is your name?");
+
+alert(`✨ Welcome ${yourName} ✨`);
+
 // Step 2 - Food choice
-// Your code goes here
+const menuSelect = parseInt(
+  prompt(`
+  What would you like to order?
+    Enter the number of your choice:
+    1. Pizza 🍕
+    2. Pasta 🍝
+    3. Salad 🥗
+    `
+  )
+);
+
+console.log("selected menu value", menuSelect)
```

Comment:
Please remove the the console.log() around the code. Generally they need to be deteled when the code has been tested and works as meant to 👍